snappy/snapp.go

```diff
@@ -1996,3 +2015,68 @@ func makeSnapHookEnv(part *SnapPart) (env []string) {
 
 	return env
 }
+
+// Fixup cleans up after some errors
+func Fixup() error {
+	// This fixes *one* issue. Move it out into a list of
+	// functions if you need to fix more than one. Keep each fixup
+	// simple.
+
+	installed, err := NewMetaLocalRepository().Installed()
```

Comment:
A short comment what this fixup is doing (at a high level) would be nice, so that my future self will have an easier time when looking at this again :)